### PR TITLE
updated update arrays page

### PR DESCRIPTION
**Pull request by kyuan-mongodb**
## Pull Request Info

Updated the CRUD > Write Operations > Update Arrays page.

I know the ticket didn't say to update the entire page, but I thought it could use an improvement in terms of wording.

I'll make a PR to each versioned branch after this gets approved.

[PR Reviewing Guidelines](https://github.com/mongodb/docs-node/blob/master/REVIEWING.md)

### Issue JIRA link:
https://jira.mongodb.org/browse/DOCSP-20598

### Snooty build log:
https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=623b62680ab54725f1946e6a

### Docs staging link (requires sign-in on MongoDB Corp SSO):
https://docs-mongodbcom-staging.corp.mongodb.com/node/docsworker-xlarge/DOCSP-20598-Dot-Notation/fundamentals/crud/write-operations/embedded-arrays/

### Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Does it render on staging correctly?
- [x] Are all the links working?
- [x] Are the staging and workerpool job links in the PR description updated?

### If your page documents a concept, does it meet the following criteria?

- [x] Target the [Jasmin persona](https://drive.google.com/file/d/14FbBOLCVxwSP6M9BK4Nz1Ir9tzxT8_02/view)
- [x] Target the [Lucas persona](https://drive.google.com/file/d/1J2vqJxo7ldv7OP_obA9Q-avf0o_ju4Lk/view)

### CODE REVIEW

File: source/fundamentals/crud/write-operations/embedded-arrays.txt [hunk 1]
```diff
@@ -28,108 +28,94 @@ for a complete list.
...
+Specifying Array Elements
+-------------------------
+
+You can specify which array elements to update using a positional
+operator. Positional operators can specify the first, all, or certain
+array elements to update. 
+
+To specify elements in an array with positional operators, use **dot
+notation**. Dot notation is a property access syntax for navigating BSON
+objects. To learn more, see :manual:`dot notation
```
Comment (terakilobyte) [inline source/fundamentals/crud/write-operations/embedded-arrays.txt:68]: issue: If there is a ref on this page, use it instead. If not, feel free to dismiss this comment.

File: source/fundamentals/crud/write-operations/embedded-arrays.txt [hunk 2]
```diff
@@ -140,81 +126,66 @@ in our update, we encounter the following error:
...
 
-Match All Array Elements
-------------------------
+Matching All Array Elements
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 To perform the update on all of the array elements of each document that
-matches your query document in your update operation, use the all
-positional operator, ``$[]``.
+matches your query document, use the all positional ``$[]`` operator.
```
Comment (terakilobyte) [inline source/fundamentals/crud/write-operations/embedded-arrays.txt:135]: issue: Two things. Throughout the page, we're saying "array elements of each document that matches your query document". I think we can safely drop the second use of 'document', e.g. "... document that matches your query".

Next, I think it's better to show the operator after giving it its name, so use the "all positional operator >operator<".

File: source/fundamentals/crud/write-operations/embedded-arrays.txt (hunk 1 above)
Comment (kyuan-mongodb) [reply source/fundamentals/crud/write-operations/embedded-arrays.txt:68]: Found their page [here](https://github.com/mongodb/docs/blob/master/source/reference/operator/update-array.txt), there's no ref

File: source/fundamentals/crud/write-operations/embedded-arrays.txt [hunk 3]
```diff
@@ -13,13 +13,13 @@ Update Arrays in a Document
...
 array update operator in your update method call. In this guide, we
-explain and show examples on usage of these operators including:
+explain and show examples on how to update the following:
 
-- :ref:`First array element match positional operator <first-match-operator>`: ``$``
-- :ref:`All array element match operator <all-match-operator>`: ``$[]``
-- :ref:`Filtered positional operator <filtered-positional-operator>`: ``$[<identifier>]``
+- :ref:`first-match-operator`
+- :ref:`All Matching Array Elemetns <all-match-operator>`
+- :ref:`Multiple Matching Array Elements <filtered-positional-operator>`
```
[the remark this answers is not in the document]
Comment (kyuan-mongodb) [reply source/fundamentals/crud/write-operations/embedded-arrays.txt:22]: Agreed, I'll update/sort of revert the intro to mention the three types of operators

**Review by terakilobyte — COMMENTED** [review]
LGTM, but I think adding the flavor to the examples as I suggested makes it look weird now, since the first example (and the other examples) aren't really identifiable/meaningful.

I'd just remove the headings from the OTP widget.

**Review by terakilobyte — APPROVED** [review]

**Pull request merged**